client/my-sites/upgrades/checkout/domain-details-form.jsx

```diff
@@ -177,6 +203,35 @@ export default React.createClass( {
 		);
 	},
 
+	renderPhoneField() {
+		const label = this.translate( 'Phone', { textOnly: true } );
+
+		if ( abtest( 'domainContactNewPhoneInput' ) === 'enabled' ) {
+			return (
+				<FormPhoneMediaInput
+					label={ label }
+					countriesList={ countriesList }
+					countryCode={ this.state.phoneCountryCode }
+					onChange={ this.handlePhoneChange }
+					{ ...omit( this.getFieldProps( 'phone' ), 'onChange' ) } />
+			);
+		} else {
```

Comment:
Fixed this but honestly, it makes the code harder to read with a conditional return. I think we should change the linter to allow if-else with only one if block.